README.rst

```diff
@@ -120,8 +127,19 @@ Optional::
     # The expiry should be longer than the window (not equal);
     # otherwise, a URL may expire before a new one is available.
     # If either value is zero or negative, then URL caching is disabled.
+    # 'public_url_cache_window': How long a public (unsigned) URL will be reused.
     ckanext.s3filestore.signed_url_expiry = 3600
     ckanext.s3filestore.signed_url_cache_window = 1800
+    ckanext.s3filestore.public_url_cache_window = 86400
+
+    # Control how long the ACL of an S3 object will be held in cache.
+    # Uploading a new file overrides this. Default is 86400.
+    ckanext.s3filestore.acl_cache_window = 2592000
+
+    # If set, then prior objects uploaded for a resource may be deleted
+    # after the specified number of days. If less than zero, nothing
```

Comment:
Please update this that this is about when resource was uploaded. Not when delete function was called.
i.e. if uploaded over 90 days ago, it will be purged if set to 90. if it was uploaded yesterday, it will be kept for 89 days. Then it is up to the configuration of the s3 bucket on how long deleted items are kept.

Reply:
I'm not sure which part of the documentation is unclear about that.